packages/plugin-vue/src/utils/descriptorCache.ts

```diff
@@ -49,3 +49,17 @@ export function getDescriptor(filename: string, errorOnMissing = true) {
 export function setDescriptor(filename: string, entry: SFCDescriptor) {
   cache.set(filename, entry)
 }
+
+export function isEqualBlock(a: SFCBlock | null, b: SFCBlock | null) {
```

Comment:
I don't think we need to move this function. Just export it from `handleHotUpdate` is fine.

Reply:
OK, done.